kodilla-patterns2/src/test/java/com/kodilla/patterns2/decorator/pizza/PizzaOrderTestSuite.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.patterns2.decorator.pizza;
+
+import org.junit.Test;
+import java.math.BigDecimal;
+import static org.junit.Assert.*;
+
+public class PizzaOrderTestSuite {
+
+    @Test
+    public void testPizzaGetCost(){
+        //Given
+        PizzaOrder pizzaOrder = new BoxPizzaOrder();
+        pizzaOrder = new Pizza(pizzaOrder, OptionPizza.CAPRICIOSA);
+        pizzaOrder = new Pizza(pizzaOrder, OptionPizza.MARGHERITA);
+        pizzaOrder = new Ingredient(pizzaOrder,ExtraIngredient.CHEESE);
+        //When
+        BigDecimal theCost = pizzaOrder.getCost();
+        //Then
+        assertEquals(new BigDecimal(54),theCost);
+        showResult(pizzaOrder);
+    }
+
+    @Test
+    public void testPizzaGetDescription(){
+        //Given
+        String expect = "Box + PARMA ( tomatoSauce, cheese, ham, pepper)" +
+                "  + Extra Double cake + Extra Ham + Extra Onion + Extra pepper";
+        PizzaOrder pizzaOrder = new BoxPizzaOrder();
+        pizzaOrder = new Pizza(pizzaOrder,OptionPizza.PARMA);
+        pizzaOrder = new Ingredient(pizzaOrder,ExtraIngredient.DOUBLE_CAKE);
+        pizzaOrder = new Ingredient(pizzaOrder, ExtraIngredient.HAM);
+        pizzaOrder = new Ingredient(pizzaOrder, ExtraIngredient.ONION);
+        pizzaOrder = new Ingredient(pizzaOrder,ExtraIngredient.PEPPER);
+        //When
+        String order = pizzaOrder.getDescription();
+        //Then
+        assertEquals(expect,order);
+        showResult(pizzaOrder);
+    }
+
+    private void showResult(PizzaOrder pizzaOrder){
+        System.out.println(pizzaOrder.getDescription() + " cost: " + pizzaOrder.getCost()+ " ZL");
+    }
```

Comment:
pamietaj prosze, ze w realnych testach to jest niepotrzebne. Bazujesz na rezultacie z assertEquals, nie printowaniu do konsoli